src/plot_api/accessibility.js

```diff
@@ -0,0 +1,71 @@
+"use strict";
+import c2mChart from "chart2music";
```

Comment:
To pass current es5 build tests, this file should be written in es5 style similar other files to `src`.

Reply:
Do we still need to be writing in es5? Won't webpack take that and give us the desired output? I'd love it if we can start accepting (and eventually converting existing code to) more modern syntax.

Reply:
For es modules, we have a config rule in place.
I did try to add a rule to transpile this file during build processes; but `npm run build` wasn't successful in the past hour.
If one could help with setting up the webpack config, it would be great :pray:

Reply:
Thank you for these tips again! I've made some attempts at this (putting chart2music and/or accessibility.js into the webpack config but struggling with "Cannot find module 'chart2music'" whenever I use require instead of import. I think I'm still missing something